### PR TITLE
[IOS-563] Polish Dapp Profile

### PR DESCRIPTION
**Pull request by marijnschilling**
img_019914a57f9e-1

- [x] When there's no header image we show a smaller header
- [x] Set the right font and spacing for the Dapp title
- [x] Set line spacing on description
- [x] I also stuck the bottom insets to the tab bar.

https://trello.com/c/mzq4eObx/36-dapp-profile-polishing
https://toshiapp.atlassian.net/browse/IOS-562

### CODE REVIEW

File: Toshi/Controllers/Dapps/Views/DappInfoView.swift [hunk 1]
```diff
@@ -138,7 +139,10 @@ final class DappInfoView: UIView {
         mainStackView.left(to: self, offset: .defaultMargin)
         mainStackView.right(to: self, offset: -.defaultMargin)
 
+        titleLabel.height(24)
```
Comment (yuliaveres) [inline Toshi/Controllers/Dapps/Views/DappInfoView.swift]: why do you constraint height for the label with Dynamic font ?

File: Toshi/Controllers/Dapps/DappViewController.swift [hunk 2]
```diff
@@ -109,7 +110,11 @@ final class DappViewController: DisappearingNavBarViewController {
         guard let path = dapp.coverUrlString else { return }
```
Comment (yuliaveres) [inline Toshi/Controllers/Dapps/DappViewController.swift]: setup no cover image here as well please

File: Toshi/Controllers/Dapps/DappViewController.swift [hunk 3]
```diff
@@ -18,7 +18,8 @@ import UIKit
 import SweetUIKit
 
 final class DappViewController: DisappearingNavBarViewController {
-    let dappConverHeaderHeight: CGFloat = 200
+    let dappCoverHeaderHeight: CGFloat = 200
+    let dappNoCoverHeaderHeight: CGFloat = 44
```
Comment (yuliaveres) [inline Toshi/Controllers/Dapps/DappViewController.swift]: this height is not enough, what about status bar ?

Comment (yuliaveres) [reply Toshi/Controllers/Dapps/DappViewController.swift]: img_0389

Comment (marijnschilling) [reply Toshi/Controllers/Dapps/DappViewController.swift]: add the safearea to it

**Pull request merged**